Wordle.Api/Wordle.Api/Controllers/LeaderboardController.cs

```diff
@@ -0,0 +1,32 @@
+using Microsoft.AspNetCore.Mvc;
+using Wordle.Api.Data;
+using Wordle.Api.Dtos;
+using Wordle.Api.Services;
+
+namespace Wordle.Api.Controllers
+{
+    [ApiController]
+    [Route("api")]
+    public class LeaderboardController : ControllerBase
+    {
+        private readonly LeaderboardService _leaderboardService;
+
+        public LeaderboardController(LeaderboardService leaderboardService)
+        {
+            _leaderboardService = leaderboardService;
+        }
+
+        [HttpGet("Leaderboard")]
+        public IEnumerable<Player> GetLeaderboard()
+        {
+            return _leaderboardService.GetLeaderboard();
+        }
+
+        [HttpPost("AddPlayer")]
```

Comment:
```suggestion
        [HttpPost("UpsertPlayer")]
```
You may not be adding a player, you might just be updating a score for an existing player. This could also be called 'UpdateScore' or just 'Score'